### PR TITLE
Replaced all the css into tailwind css on ProgressIndicator component

### PR DESCRIPTION
**Pull request by subhajit20**
# Description of changes

Replaced all the css into tailwind css on ProgressIndicator component
Issue - #1725 

Before:
Before

After:
After

# Issue Resolved

Fixes #NA

### CODE REVIEW

File: components/ProgressIndicator/ProgressIndicator.js [hunk 1]
```diff
@@ -31,7 +30,7 @@ export default function ProgressIndicator({ stepNumber, totalSteps }) {
   const percentageCompleted = (stepNumber / totalSteps) * 100;
 
   return (
-    <div className={styles.ProgressIndicator}>
+    <div className="flex flex-col text-center my-6 mx-0 items-center">
```
Comment (kylemh) [inline components/ProgressIndicator/ProgressIndicator.js:33]:
```suggestion
    <div className="flex flex-col my-6 mx-0 items-center">
```

**Comment by kylemh** [comment]
Because of the broken tests, I just realized in #1766 that we haven't been updating snapshot tests.

I'm glad you started trying the UI in Storybook alongside your changes to verify them! In addition, please remember to run `yarn test -u`

We do run tests as part of a pre-commit hook, but perhaps you've been skipping those?

**Comment by subhajit20** [comment]
> Because of the broken tests, I just realized in #1766 that we haven't been updating snapshot tests.
> 
> I'm glad you started trying the UI in Storybook alongside your changes to verify them! In addition, please remember to run `yarn test -u`
> 
> We do run tests as part of a pre-commit hook, but perhaps you've been skipping those? Unsure though... You _have_ been running them, maybe it's because the automatic tailwind class sorting? Unsure.

After making changes I always run yarn jest components/demoComponent/ -u that modifies all the snapshots required.
but did not know about this yarn test staff.
I will do that. thanks

**Comment by kylemh** [comment]
Instead, simply run `yarn test -u` and it will see if _any_ snapshot requires an update.

**Comment by subhajit20** [comment]
> Instead, simply run `yarn test -u` and it will see if _any_ snapshot requires an update.

done

**Comment by subhajit20** [comment]
hey @kylemh would you merge this pr now?

**Pull request merged**